### ISSUE TITLE
fails at faunadb setup

### ISSUE THREAD

**Issue by BhaskaranR** [pbteja1998/nextjs-starter#9]
good work i am trying to setup db and i am asked to input the provide a FaunaDB admin key or set the  FAUNA_ADMIN_KEY  environment and restart the tool.

how do i set the admin key? i created a key for the database but that doesn't seem to work.

**Comment by pbteja1998** [comment]
@BhaskaranR can you tell me how you created the key? You should first create a database in Fauna, then go to Security Tab, and create a key there. After that, you can run `yarn setup-db`. Please let me know if you face any more problems.

**Comment by BhaskaranR** [comment]
I don't have fauna dashboard local setup. the link mentioned is not available. i created the db in the docker shell.
```
fauna add-endpoint http://localhost:8443/ --alias localhost --key secret
fauna create-database development_db --endpoint=localhost
fauna create-key development_db --endpoint=localhost
```
then i copied this key to .env
```
FAUNADB_SECRET=secret
FAUNA_ADMIN_KEY=fnAEG4FkPgACAPuBn62LO07rjVDjj25_BtRsk2o0
```

**Comment by pbteja1998** [comment]
@BhaskaranR I just updated the README with instructions to use Fauna Dev docker container.

Check it out at https://github.com/pbteja1998/nextjs-starter#how-to-run-fauna-locally

TL;DR;
When you are using Fauna Docker container and running it locally, your FAUNADB_SECRET AND FAUNA_ADMIN_KEY, both are same and equal to `secret`

**Comment by BhaskaranR** [comment]
it still asks me for the admin key
![image](https://user-images.githubusercontent.com/11024551/114978069-8f577800-9e3d-11eb-9234-e830c8826615.png)

**Comment by pbteja1998** [comment]
Are you running `yarn setup-docker-db` or `yarn setup-db`?

**Comment by pbteja1998** [comment]
Also, can you please download the latest version of the template? I made a few changes today. You seem to be using the old version.

**Comment by BhaskaranR** [comment]
works good now. thanks for all the help!

**Comment by pbteja1998** [comment]
Awesome. Very glad to hear.
